go.mod

```diff
@@ -126,3 +126,5 @@ require (
 // at that tag it no longer provides github.com/cactus/go-statsd-client/statsd
 // thereby removing the import.
 replace github.com/cactus/go-statsd-client => github.com/cactus/go-statsd-client v3.2.1+incompatible
+
+replace go.temporal.io/api => ../api-go
```

Comment:
This and the conflicts will be fixed when https://github.com/temporalio/api/pull/134 is merged.